pages/analysis.md

```diff
@@ -1,15 +1,132 @@
+{% include nav.html %} {% include css.html %}
+
 # Analysis
+---
+	
+The visualization on this page represents the documented interference
+on the slave trade by privateers and pirates between 1699-1827. The primary 
+geographical focus is in the Caribbean and West Indies region. These regions 
+are where the most slave trade activity took place. The original hypothesis 
+for this research is that privates and privateers had a significant effect on 
+the slave trade. Pirates and privateers caused fiscal strains on the slave t
+rade market and economically effected countries that participated in the slave 
+trade. As per the documented information, the slave trade was extremely expansive
+that the incursion by privateers had little to no effect on the market itself. 
+
+The images do raise questions to the effects privateers and pirates had on smaller
+countries. These smaller states would not have had such availability to the slave 
+trade if not provided by the captive ships. The images below demonstrate the 
+disparities and theory.
+
+---
+
+![image]({{ site.baseurl}}/assets/images/1.%20Origin%20Ports.png)
+	
+This image represents most of the documented ports where slave 
```

Comment:
One thing that can make it a little easier to discuss images is to number them somehow. You can just say `Figure 1` below the image, and then in your main text say `Figure 1 shows...` or `Some claim (Figure 1)`.

One way to do this is with the <caption> tag, that just resizes the text to look appropriate for an image caption. I added a suggested change below.